org-code-javabuilder/validation/src/main/java/org/code/validation/NeighborhoodLog.java

```diff
@@ -0,0 +1,96 @@
+package org.code.validation;
+
+public class NeighborhoodLog {
```

Comment:
NIT: Would `NeighborhoodLogAnalyzer` be more accurate?

Reply:
probably, this class will be used by Levelbuilders though so I'm inclined to keep the names shorter